internal/store/postgres/appeal_repository.go

```diff
@@ -138,6 +85,18 @@ func (r *AppealRepository) Find(ctx context.Context, filters *domain.ListAppeals
 	return records, nil
 }
 
+func (r *AppealRepository) GetAppealsTotalCount(ctx context.Context, filter *domain.ListAppealsFilter) (int64, error) {
+	db := r.db.WithContext(ctx)
+	db = applyAppealFilter(db, filter)
+
+	var count int64
+	if err := db.Model(&model.Appeal{}).Count(&count).Error; err != nil {
+		return 0, err
+	}
+
+	return count, nil
```

Comment:
this would be simpler
```suggestion
	var count int64
	err := db.Model(&model.Appeal{}).Count(&count).Error
	return count, err
```

Reply:
yupyup okay